package.json

```diff
@@ -61,9 +62,10 @@
     "pub-sub-es": "^2.0.1",
     "react-checkbox-tree": "^1.7.3",
     "react-color": "^2.13.8",
-    "react-grid-layout": "^0.16.6",
-    "react-simple-code-editor": "^0.9.10",
-    "react-sortable-hoc": "^1.10.1",
+    "react-grid-layout": "1.3.4",
+    "react-resizable": "^3.0.5",
```

Comment:
Did we have to add a new dependency? I don't see this from before.

Reply:
I see something from there being imported in HiGlass.scss. Whether we need to specify it as a dependency is unclear. We can try removing it once all the tests pass.

Reply:
Was able to remove this one.